lib/main.rb

```diff
@@ -0,0 +1,10 @@
+require_relative 'parse'
+
+path = '../log/games.log'
+
+instance = Parse.new(path)
+
+if(File.exist?(path))
```

Comment:
@wesjrock não seria melhor fazer essa verificação no seu initialize? o que aconteceria se esse código for executado em outro lugar? por exemplo nos testes?